Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12.13.0-alpine
+WORKDIR /app  
```

Comment:
To keep the example simple, let's not have a docker file and use the one on KintoHub.